Kava/Kava/Source/PageObjects/Infrastructure/Elements/CollectionView.swift

```diff
@@ -65,7 +65,7 @@ open class CollectionView<TResultBlock : Block> : Element {
     }
     
     open func scrollForwardUntil<TCustomResultBlock : Block>(_ block: @escaping (() -> Bool), timeout: TimeInterval = 5, result: TCustomResultBlock.Type, constructingBlock builder: (() -> TCustomResultBlock)? = nil) -> TCustomResultBlock {
-        let predicate = NSPredicate { (_: Any, _: [String : Any]?) -> Bool in
+        let predicate = NSPredicate { (_, _) -> Bool in
```

Comment:
The return type/number of arguments are still specified. See my original suggestion